src/main/java/boombotroz/Ui.java

```diff
@@ -0,0 +1,134 @@
+package boombotroz;
+import java.time.LocalDate;
+
+/**
+ * Deals with interactions with the user.
+ */
+public class Ui {
+
+    /**
+     * Throws exception when input invalid.
+     */
+    public void invalidInput() throws BoomException {
+        throw new BoomException("That's nonsense !!");
+
+    }
+
+    /**
+     * Throws exception when no task details.
+     *
+     * @param taskInput task details.
+     * @throws BoomException If no task details given.
+     */
+    public void emptyTask(String taskInput) throws BoomException {
```

Comment:
Maybe you can come up with a verb for this method's name that describes the intended action of this method?